rudder-web/src/main/resources/rudder-apache-webapp-common.conf

```diff
@@ -4,8 +4,6 @@
   Header add X-Chrome-Exponential-Throttling "disable"
 </LocationMatch>
 
-DocumentRoot /var/www
-
 # Prevent the apache logs from beeing cluttered by 404 errors
 # due to a missing robots.txt file.
 Alias /robots.txt /opt/rudder/share/load-page/robots.txt
```

Comment:
ErrorDocument shoud work properly in relay and we should avoid sending it if it is called in a relay api